yellow-paper/docs/decentralization/actors.md

```diff
@@ -0,0 +1,182 @@
+---
+title: Nodes and Actors
+---
+
+To analyze the suitability of different node types for various actor types in the system, let's break down the node types and actor types, considering the characteristics, preferences, abilities, and potential motivations of each actor. 
+This will help determine which actors are likely to run which nodes and how they might interact with the Private eXecution Environment (PXE).
+
+## Background
+
+Before diving into Aztec specific data, we take a look at general blockchain nodes. 
+It should help us have a shared base, which will later be useful for comparing the Aztec setups, where multiple nodes are involved due to its construction.
+
+### Node Types
+
+The below node types are ordered in resource requirements, and exists for most blockchains. 
+All the nodes participate in the peer-to-peer (p2p) network but with varying capacity.
+
+1. **Light Node**:
```

Comment:
I'm still unsure of the purpose of a 'light' node. With regards to the p2p network, I wasn't planning on broadcasting the block headers, nodes will just read them from L1 which is the source of truth. The p2p network only needs to propagate transactions.

Reply:
Light nodes are to not blindly trust something like infura. 
The section is not worrying around the specific type of blockchain, but general. 

But since the l1 contract we have is a validating light node, you can use it as you p2p connection.